ray_on_golem/server/services/yagna.py

```diff
@@ -5,13 +5,17 @@
 from pathlib import Path
 from typing import Optional
 
+import aiohttp
+from yarl import URL
+
 from ray_on_golem.exceptions import RayOnGolemError
 from ray_on_golem.server.settings import YAGNA_APPKEY
 from ray_on_golem.utils import run_subprocess
 
 logger = logging.getLogger(__name__)
 
 YAGNA_APPNAME = "ray-on-golem"
+YAGNA_API_URL = URL("http://127.0.0.1:7465")
```

Comment:
don't we want to allow this to be overridden? by an environment variable perhaps?